### ISSUE TITLE
'BatchNorm2d' object has no attribute '_non_persistent_buffers_set'

### ISSUE THREAD

**Issue by jeffin07** [JirongZhang/DeepHomography#23]
Hai @JirongZhang,

When I try to run test.py I encountered this error can you please help me what is that I'm missing here ?.
Pytorch version: 1.6.0

**Comment by frozenbananana** [comment]
Hi, may be you need a lower version of Pytorch
